### DISCUSSION TITLE
Chia db and wallet

### DISCUSSION THREAD

**Discussion by Pjeronski** [Chia-Network/chia-blockchain#8488] [Support]
Hi guys,
I'd like to ask you how to move db and wallet files from sd card to external hard drive in my RPi4? Those files are already quite big and are still growing and I have still some free space on my drives and I'd like to utilize this instead of buying bigger sd card. I tried already to move those files (db and wallet folders) to external drive and to modify config.yaml (I changed directories where it was needed, for example: from `database_path: db/blockchain_v1_CHALLENGE.sqlite` to ` database_path: /media/pi/Segate drive/new/db/blockchain_v1_CHALLENGE.sqlite`). I can even see the last block height in the wallet tab in my gui (for me it is 863,589), but the full node tab is loading all the time. You can see my log attached. 
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/7173418/debug.log)

Someone has a solution how to change it?
Thanks in advance!

Peace
Adam

**Comment by loppefaaret** [comment]
perhaps you could try closing down Chia, clean out the `debug.log` and then proceed to start chia. i need more logging from the full_node in the logs

**Comment by DFC72** [comment]
Was this resolved? Is all there is to it is changing the config.yaml file?

**Reply by loppefaaret** [reply]
following what the OP did, to change the location of the databases is all that is needed - note that both the "full_node:" and "wallet:" section has this entry, for their correspondant databases
